src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-information/child-information.js

```diff
@@ -0,0 +1,68 @@
+import { genericSchemas } from '../../../generic-schema';
+import { suffixes } from '../../../constants';
+import { childInfo } from './helpers';
+import { isChapterFieldRequired } from '../../../helpers';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import _ from 'lodash/fp';
```

Comment:
There's been a slew of pull requests recently to remove lodash or lodash/fp imports from files, and instead just import individual methods where possible. I think it'd be best if we followed that convention as well.

Checkout the PRs here - 
https://github.com/department-of-veterans-affairs/vets-website/pull/11894
https://github.com/department-of-veterans-affairs/vets-website/pull/11896
https://github.com/department-of-veterans-affairs/vets-website/pull/11897

Reply:
Good to know! Updated this